client/executor/src/wasm_runtime.rs

```diff
@@ -86,17 +86,18 @@ impl RuntimesCache {
 	///
 	/// # Parameters
 	///
-	/// `ext` - Externalities to use for the runtime. This is used for setting
-	/// up an initial runtime instance.
+	/// `ext` - Externalities to use for the runtime.
```

Comment:
I think we need `ext` for executing `Core_version` call here...

```suggestion
	/// `ext` - Externalities to use for the getting the runtime's version call.
```